fiftyone/multimodal/ingest/__init__.py

```
@@ -6,37 +6,42 @@
 
-from fiftyone.core import storage
+import fiftyone as fo
+from .read import _get_scene_inventories
+from fiftyone.multimodal.db.mongo import MongoAdapter
 
+__all__ = ["ingest_files"]
 
-def _get_scene_inventories(filepaths, *, adapter):
+
+def ingest_files(
+    dataset: fo.Dataset, filepaths: list[str], *, adapter, manifest
```

Ruff (0.15.12)

[warning] Unused function argument: `manifest`

(ARG001)